src/amplitude-client.js

```diff
@@ -1684,7 +1692,7 @@ AmplitudeClient.prototype.sendEvents = function sendEvents() {
       //  here.
       // }
     } catch (e) {
-      // utils.log('failed upload');
+      // utils.log.error('failed upload');
```

Comment:
what's this line for if commented out?

Reply:
Was commented out before, didn't want to touch change it 😂

Reply:
Just uncommented it out